logs/logger.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(10)
+formatter = logging.Formatter("%(levelname)s    %(asctime)s    %(pathname)s    %(message)s",
```

Comment:
Apply the Python formatter `black` to the whole application.
https://black.readthedocs.io/en/stable/getting_started.html#basic-usage